extensions/amp-autocomplete/0.1/test/test-amp-autocomplete.js

```diff
@@ -232,6 +232,52 @@ describes.realWin(
       });
     });
 
+    it('getRemoteData_() should proxy XHR to viewer', () => {
```

Comment:
Since this is a new test, you should use async/await instead of promises. So:

```javascript
it('blah', async () => {
  await impl.getRemoteData_();
  expect(impl.ssrTemplateHelper_.ssr).to.be.calledOnce;
});
```